.gitignore

```diff
@@ -3,4 +3,5 @@
 /build/daemon.zip
 /static/daemon/lbrynet*
 /static/locales
+/.idea
```

Comment:
Please keep user-dependent rules in your global `.gitignore` file: https://help.github.com/articles/ignoring-files/